app/api/ringle_music/v1/user_api.rb

```diff
@@ -0,0 +1,45 @@
+module RingleMusic
+    module V1
+        class UserApi < Grape::API
+            version 'v1', using: :path
+            format :json
+            prefix :api
+
+            resource :users do
+
+                get :info do
+                    begin
+                        UserService::UserGetter.call(request: request)
+                    rescue ActiveRecord::RecordNotFound => e
+                        return {success: false, message: "User Not Found"}
+                    rescue => e
+                        return {success: false, message: e.message}
+                    end
+
+                end
+                 
+            
+                params do 
+                    requires :user_name, type: String
+                    requires :email, type: String 
+                    requires :password, type: String
+                end
+                post "signup" do
+
+                    UserService::UserSignup.call(user_name: params[:user_name], email: params[:email], password: params[:password])
+
+                end
+
+                params do
+                    requires :email, type: String
+                    requires :password, type: String
+                end
+                post "signin" do
+
+                    UserService::UserSignin.call(email: params[:email], password: params[:password])
```

Comment:
모든 params를 넘기는 것이 목적이라면
```suggestion
                    UserService::UserSignin.call(params.symbolize_keys)
```
이런 방식으로 넘길 수 있습니다.